.github/workflows/update-plugin-list.yml

```
@@ -27,12 +27,20 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: "3.8"
+          cache: pip
+      - name: requests-cache
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/pytest-plugin-list/
+          key: plugins-http-cache-${{ github.run_id }} # Can use time based key as well
```

Did you mean "could" here?

```suggestion
          key: plugins-http-cache-${{ github.run_id }} # Could use time based key as well.
```